src/components/sbb-link/sbb-link.scss

```diff
@@ -22,12 +22,15 @@
 .sbb-link {
   display: flex;
   align-items: center;
-  gap: var(--sbb-spacing-fixed-1x);
   color: var(--text-color-normal);
   text-decoration: none;
   word-break: break-word;
   fill: currentcolor;
 
+  &[tabindex='-1'] {
+    pointer-events: none;
+  }
+
```

Comment:
Why is this necessary?

Reply:
I see that disabled link also visually does not exist in at the moment (hover styles and href still active). I think we will have to discuss this topic and introduce it in a separate PR. @kyubisation ?

Reply:
Yes, we can ignore disabled state for now 👍

Reply:
@kyubisation But for now we don't include this change right? because it would be only the half of necessary work

Reply:
Yes, let's remove this for now. We will discuss disabled link states in the future with Manuel.

Reply:
removed it